compiler/crates/relay-lsp/tests/hover/fixtures/testing.graphql

```diff
@@ -0,0 +1 @@
+# Just testing to ensure that CI fails
```

Comment:
Added temporarily to validate CI changes.

Reply:
Validated that this causes CI to fail:

<img width="1210" alt="Screenshot 2023-01-09 at 2 07 47 PM" src="https://user-images.githubusercontent.com/162735/211418243-f506750e-770b-476b-8c85-4c4d04cf9676.png">